### PR TITLE
[DOCFIX] Fix POSIX API doc for umount command

### PR DESCRIPTION
**Pull request by LuQQiu**
### What changes are proposed in this pull request?

Fix POSIX API doc for umount command

### Why are the changes needed?
Have detailed instructions for how to umount fuse
### Does this PR introduce any user facing changes?
alluxio-fuse CLI command

### CODE REVIEW

**Comment by LuQQiu** [comment]
TODOs for future PRs
1) Docker how to umount external Fuse mount point
2) Kubernetes how to do soft umount first and then do hard umount after

**Review by ssz1997 — CHANGES_REQUESTED** [review]
Mostly concern about the `sudo`. Also could you update the env variable here? https://github.com/Alluxio/alluxio/blob/08eac2606fb0d02b0dba7ed1196ab3f25b37c533/integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml#L112 Thanks

File: integration/fuse/bin/alluxio-fuse [hunk 1]
```diff
@@ -160,24 +161,27 @@ umount_fuse() {
     echo "${success_msg}"
     return 0
   fi
-  if umount -f "${mount_point}" > ${ALLUXIO_LOGS_DIR}/fuse.out 2>&1; then
+  if sudo umount -f "${mount_point}" > ${ALLUXIO_LOGS_DIR}/fuse.out 2>&1; then
```
[the remark this answers is not in the document]
Comment (LuQQiu) [reply integration/fuse/bin/alluxio-fuse:164]: good point, umount -f needs sudo priviledge both in our docker, centos, and local mac

**Review by ssz1997 — APPROVED** [review]
LGTM

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**